tests/functional/spec/thank_you.spec.js

```diff
@@ -52,20 +52,19 @@ describe("Thank You Default View Response Enabled", () => {
       expect(browser.getUrl()).to.contain(ThankYouPage.pageName);
```

Comment:
So these will test with JS, I don't think we have any tests to ensure the content is as expected for non-js version?. Previously it didn't matter since the content was the same between the two.

Reply:
it appears that once cannot update the browser option during runtime. so can't test this option. If we disable before launching surveys then we can't do any survey.

Reply:
Not sure what you mean. The Non JS test would live as an integration test which has no JS. Runner doesn't require JS to work.

Reply:
added integration test